client/landing/gutenboarding/onboarding-block/features/style.scss

```diff
@@ -31,7 +31,7 @@
 	margin: -12px;
 }
 
-.features__item.components-button.is-tertiary {
+.features__items > button.features__item.components-button.is-tertiary {
```

Comment:
I'm still getting used to the repo, but this feels a bit hacky — it looks like you increased the specificity of this rule so that you could override the `.edit-post-visual-editor .components-button.is-tertiary` selector.
Would there be any other alternative?

Reply:
That's exactly right. There might be a different way to increase the specificity but this one works alright. This stems from the fact that `.components-button.is-tertiary` has relatively high specificity to begin with.

Reply:
I see. It just felt weird that a button used while creating a new _site_ is influenced by a class called `.edit-post-visual-editor`.  But this is something to be discussed separately from this PR, thank you for the explanation!